Solutions/STM32F4DISCOVERY/TinyBooter/TinyBooter.proj

```diff
@@ -23,6 +23,8 @@
         <ExtraTargets>$(ExtraTargets);CompressBin</ExtraTargets>
         <ScatterFileDefinition>scatterfile_bootloader_$(COMPILER_TOOL).$(SCATTER_EXT)</ScatterFileDefinition>
         <EXEScatterFileDefinition>scatterfile_bootloader_$(COMPILER_TOOL).$(SCATTER_EXT)</EXEScatterFileDefinition>
+        <!-- newlib-nano does not support long-long and long-double formatting, use standard newlib instead -->

```

Comment:
What is it in Tinybooter that is requiring long-long and long-double formatting? Perhaps we should fix/remove that so we don't need to do this and can use newlib-nano?